safe/report/impact_report.py

```diff
@@ -18,15 +18,13 @@
                  'Disaster Reduction')
 import os
 import logging
-
-# noinspection PyUnresolvedReferences
 from qgis.core import (
     QgsComposition,
     QgsRectangle,
     QgsMapLayer,
     QgsComposerHtml,
     QgsComposerFrame)
-
+# Whoaa this is ugly can we get rid of it?
```

Comment:
I'm going to leave it for now until I understand why it was added there - can fix that another time...